src/lib/state/swap.ts

```diff
@@ -24,5 +24,6 @@ export const swapAtom = atomWithImmer<Swap>({
 })
 
 export const independentFieldAtom = pickAtom(swapAtom, 'independentField')
-export const integratorFeeAtom = pickAtom(swapAtom, 'integratorFee')
-export const amountAtom = pickAtom(swapAtom, 'amount')
+
+// If set to a transaction hash, displays that transaction's status.
+export const pendingTxHashAtom = atom<string | undefined>(undefined)
```

Comment:
Should this be something more clear? Maybe `txHashToDisplay`?

Reply:
yeah its a little confusing to me because multiple can be pending - do we want to always show the oldest txn or the most recent - could be newestTxHash